src/setup/node.rs

```diff
@@ -124,6 +146,7 @@ impl NodeBuilder {
         let node = self.start_node();
         wait_for_start(node.config.local_addr).await;
 
+        self.meta = NodeMetaData::new(setup_path)?; // Reset args
```

Comment:
Why?

Reply:
Basically we need to do `meta.args.clear()` here.
Since we need to clear any previously used rippled.cfg on lines 143-144:
```
    self.meta.start_args.push("--conf".into());
    self.meta.start_args.push(rippled_cfg_path.into());
```

But we cannot simply call `clear()` because we would lose any arguments that were provided by the config.toml file:
```
        // Read the args (which includes the start command at index 0).
        let args_from = |command: &str| -> Vec<OsString> {
            command.split_whitespace().map(OsString::from).collect()
        };

        // Separate the start command from the args list.
        let mut start_args = args_from(&config_file.start_command);
        let start_command = start_args.remove(0);
```

So we need to clear the `args` that we added but keep the `args` we received from above.
The best solution I found was to recreate the `NodeMetaData` object.